docs/source/use_cases/analyze_reads/calculate_coverage.rst

```diff
@@ -0,0 +1,227 @@
+Calculate Coverage
+===========
+
+.. toctree::
+   :maxdepth: 2
+
+.. contents::
+
+This pipeline calculates mean read depth coverage for a given dataset or set of read group sets
+and writes the results to Annotations in a new AnnotationSet using the Genomics API.
+
+For each "bucket" in the given input references, this computes the average coverage (rounded to
+six decimal places) across the bucket that 10%, 20%, 30%, etc. of the input read group sets have
+for each mapping quality of the reads (<10:Low(L), 10-29:Medium(M), >=30:High(H)) as well as
+these same percentiles of read group sets for all reads regardless of mapping quality (Mapping
+quality All(A)).
+
+There is also the option to change the number of quantiles accordingly (numQuantiles = 5 would
+give you the minimum read group set mean coverage for each and across all mapping qualities, the
+25th, 50th, and 75th percentiles, and the maximum of these values).
+
+The pipeline is implemented on `Google Cloud Dataflow`_.
+
+Setup Dataflow
+--------------
+
+.. include:: ../../includes/dataflow_setup.rst
+
+.. include:: ../../includes/dataflow_on_gce_setup.rst
+
+Create Output Dataset
+---------------------
+
+In order to run this pipeline, you must create a dataset using the Genomics API that the pipeline
+can output its AnnotationSet and Annotations to.  If you already have a dataset in which you have
+write access, you may use its id.  If not, you can do the following to create one:
+
+#. Go to `this link <https://developers.google.com/apis-explorer/#p/genomics/v1beta2/genomics.datasets.create>`_ in the Genomics API Explorer for creating datasets.
+#. Put your Google Cloud Platform project number in the ``projectNumber`` field of the request body.
+#. Decide whether or not to make your dataset public by selecting or deselecting the ``isPublic`` field.
+#. Give your dataset a name in the ``name`` field.
+#. Turn on authorizing requests in the upper right corner of the window.
+#. Press Execute.
+
+The ``id`` that is generated in the response output is the output dataset id you should use when running
+the pipeline.
+
+
+Run the pipeline
+----------------
+
+The following command will calculate the mean coverage as described above for a given genomic
+region, using a bucket width of 1024 (in this case one bucket output) and the 1000 Genomes
+dataset id:
+
+.. code-block:: shell
+
+  java -cp target/google-genomics-dataflow*.jar \
+    com.google.cloud.genomics.dataflow.pipelines.CalculateCoverage \
+    --project=YOUR-GOOGLE-CLOUD-PLATFORM-PROJECT-ID \
+    --stagingLocation=gs://YOUR-BUCKET/dataflow-staging \
+    --secretsFile=/PATH/TO/YOUR/client_secrets.json \
+    --references=1:552960:553984 \
+    --bucketWidth=1024 \
+    --inputDatasetId=10473108253681171589 \
+    --outputDatasetId=YOUR_OUTPUT_DATASET_ID
+
+This can take several minutes to run.  You can check your results by using the Genomics API
+Explorer (make sure to authorize yourself!):
```

Comment:
Remove the parenthetical note and add authorization as an explicit step as you have above.